release/__init__.py

```diff
@@ -803,7 +808,7 @@ def main():
     options = parser.parse_args()
     if not hasattr(options, 'action'):
         parser.print_help()
-        print("ERROR: Must use a subcommand")
+        logger.error("Must use a subcommand")
```

Comment:
this needs to print out `ERROR:` / something similar for basic human CLI usage still. Same for the "Release configuration file" one below.